src/app/workflow/permissions/permissions.component.html

```diff
@@ -23,43 +23,40 @@
             <mat-icon matChipRemove *ngIf="owners.length > 1">cancel</mat-icon>
           </mat-chip>
           <input placeholder="Owners"
-                 [disabled]="updating"
                  [matChipInputFor]="ownerlist"
                  [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
                  [matChipInputAddOnBlur]="addOnBlur"
-                 (matChipInputTokenEnd)="addOwner($event)"/>
+                 (matChipInputTokenEnd)="add($event, 'OWNER')"/>
```

Comment:
You can use enums in html instead of a string

Reply:
Ah cool, did not know that. Pushed new commit.